src/pyfme/utils/input_generatorpy.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+"""
+Python Flight Mechanics Engine (PyFME).
+Copyright (c) AeroPython Development Team.
+Distributed under the terms of the MIT License.
+
+<Inputs generator>
+-------------
+<Provides some typical inputs>
+< ... >
+"""
+import numpy as np
+
+
+def step(t_init, T, A, time, offset=0, var=0):
+
+    """ step input
+
+    Parameters
+    ----------
+
+    t_init : float
+             time initial to start (s)
+    T : float
+        time while input is running (s)
+    A : float
+        peak to peak amplitude (rad) ### Should I choose other units?
+    t : array_like
+        time simulation vector (s)
+    offset : float
+    var : array_like
+          vector which contains previous perturbations
+
+    Returns
+    -------
+    step_input : array_like
+    """
+
+    step_input = np.zeros_like(time)
+    step_input[(time >= t_init) & (time <= t_init + T)] = A + offset
+    step_input = step_input + var

```

Comment:
añadiría un checkeo para ver que la longitud de var sea la misma que la del vector de tiempos. Además, que si no es cero, se puede inicializar el `step_input = var + offset`.
En la cabecera, habría que poner como valor por defecto `var=None` y checkear con un:
`if var is not None:` Esto en principio ahorraría alguna operación. Qué te parece?